{"name": "test_02", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "def test_02():\n        print('test_02')\n>       assert 1 == 2\nE       assert 1 == 2\n\n..\\test_case\\test_case_01.py:9: AssertionError"}, "attachments": [{"name": "stdout", "source": "058cd1f5-d4f4-4fb6-9b86-bbb7e7f12c48-attachment.txt", "type": "text/plain"}], "start": 1593010066298, "stop": 1593010066299, "uuid": "8c9194f3-263a-4d6f-867d-6c2ff97e9551", "historyId": "67a9398156540bebfc41a2248395b49b", "testCaseId": "12ca7a6a198dbb9f75ffce63b5a0c13d", "fullName": "test_case.test_case_01#test_02", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_case_01"}, {"name": "host", "value": "DESKTOP-6B7PLFO"}, {"name": "thread", "value": "6896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_case_01"}]}